neurovault/apps/users/forms.py

```diff
@@ -16,20 +17,35 @@ def save(self, commit=True):
         if commit:
             user.save()
         return user
-    
-class UserEditForm(UserChangeForm):
```

Comment:
@chrisfilo I have to change inheritance to plain ModelForm, because I see no other way to discard the read-only password field.